azure-ts-aks-multicluster/README.md

```diff
@@ -0,0 +1,66 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Multiple Azure Kubernetes Service (AKS) Clusters
+
+This example demonstrates creating multiple Azure Kubernetes Service (AKS) clusters in different regions and with
+different node counts. Please see https://docs.microsoft.com/en-us/azure/aks/ for more information about AKS.
+
+# Prerequisites
+
+Ensure you have [downloaded and installed the Pulumi CLI](https://pulumi.io/install).
+
+We will be deploying to Azure, so you will need an Azure account. If you don't have an account,
+[sign up for free here](https://azure.microsoft.com/en-us/free/).
+[Follow the instructions here](https://pulumi.io/install/azure.html) to connect Pulumi to your Azure account.
+
+# Running the Example
+
+After cloning this repo, `cd` into it and run these commands.
+
+1. Create a new stack, which is an isolated deployment target for this example:
+
+    ```bash
+    $ pulumi stack init
+    ```
+
+2. Set the required configuration variables for this program:
+
+    ```bash
+    $ pulumi config set azure:environment public
+    $ pulumi config set password --secret [your-cluster-password-here]
+    $ ssh-keygen -t rsa -f key.rsa
+    $ pulumi config set sshPublicKey < key.rsa.pub
```

Comment:
It'd be good to mention setting the required `resourceGroupLocation` config value here as well.